modules/homePage/component/v2/SecondJumbotron/FirstSectionImage.tsx

```diff
@@ -0,0 +1,11 @@
+import { useWindowSize } from "../../../../../utils/useWindowSize";
+
+function FirstSectionImage() {
+    const size = useWindowSize();
+    if (size.width && size.width > 768) {
+        return <div className="-mx-16 h-[800px] w-screen bg-contain bg-left bg-no-repeat" style={{ backgroundImage: `url(/assets/images/homepage/section-1.svg)` }} />;
+    }
+    return <div className="-mx-6 mt-8 h-[400px] w-screen bg-center bg-no-repeat" style={{ backgroundImage: `url(/assets/images/homepage/section-1-small.svg)` }} />;
```

Comment:
### `FirstSectionImage, SecondSectionImage, ThirdSectionImage` Components
We can use `bg-[url(/assets/images/homepage/section-1.svg)]` in `className` instead of using `style` as props.
https://tailwindcss.com/docs/background-image --> in `arbitrary value` section at the bottom of the page

Reply:
ohh it's not used again since we change images

my bad will delete this component